tests/testthat/sch_prov/prov_level_diff_codes.csv

```diff
@@ -0,0 +1,8 @@
+time_period,time_identifier,geographic_level,country_code,country_name,provider_name,provider_ukprn,apps_level,ssa,starts,achievements
+201920,Academic year,Provider,E92000001,England,AWE PLC,10008081,Advanced Apprenticeship,"Construction, Planning and the Built Environment",30,~
+201920,Academic year,Provider,E92000001,England,AWE PLC,10008082,Total,Total,80,50
+201920,Academic year,Provider,E92000001,England,AWE PLC,10008083,Higher Apprenticeship,Total,~,~
+201920,Academic year,Provider,E92000001,England,AWE PLC,10008081,Advanced Apprenticeship,Total,80,50
```

Comment:
I think we wanted to test multiple names rather than multiple codes - worth adding another file/check for multiple names on the same code as that's the more common case.